### PR TITLE
[TTAHUB-2218] Update topics and order by alpha

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

1. Rename "Ongoing Monitoring Management System" to "Ongoing Monitoring and Continuous Improvement".
2. Update all historical topics to use the new "Ongoing Monitoring and Continuous Improvement".
3. Add new topic "Fatherhood / Male Caregiving".
4. Alphabetize topics in the filters, ar, and tr pages.

## How to test

Verify the above.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2218

### Every PR

- [x] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: packages/common/src/index.js [hunk 1]
```diff
@@ -55,14 +55,15 @@ const TOPICS = [
...
   'Family Support Services',
+  'Fatherhood / Male Caregiving',
   'Fiscal / Budget',
   'Five-Year Grant',
   'Home Visiting',
   'Human Resources',
   'Leadership / Governance',
   'Learning Environments',
   'Nutrition',
-  'Ongoing Monitoring Management System',
```
Comment (thewatermethod) [inline packages/common/src/index.js:65]: Does it still make sense to use this list instead of fetching topics from the backend?

Comment (AdamAdHocTeam) [reply packages/common/src/index.js:65]: We can create a new ticket to cleanup all the various spots. This will at least get them back to a working state.

Comment (thewatermethod) [reply packages/common/src/index.js:65]: Reminder to create that ticket

File: src/migrations/20231019233102-cleanup-ongoing-monitoring-topic.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      const oldTopicId = 127;
+      const newTopicId = 130;
+
+      await queryInterface.sequelize.query(`
```
Comment (thewatermethod) [inline src/migrations/20231019233102-cleanup-ongoing-monitoring-topic.js:15]: I'm confused as to when we use the "mapTo" column and when we don't

Comment (AdamAdHocTeam) [reply src/migrations/20231019233102-cleanup-ongoing-monitoring-topic.js:15]: The column 'mapTo' should point to the id of the topic that is replacing it. Although I don't think we are doing a lookup anywhere via this column.

image

File: packages/common/src/index.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply packages/common/src/index.js:65]: [TTAHUB-2236](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2236)

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**